docs/topics/feed-exports.rst

```diff
@@ -211,38 +211,65 @@ Settings
 
 These are the settings used for configuring the feed exports:
 
- * :setting:`FEED_URI` (mandatory)
- * :setting:`FEED_FORMAT`
+ * :setting:`FEEDS` (mandatory)
+ * :setting:`FEED_EXPORT_ENCODING`
+ * :setting:`FEED_STORE_EMPTY`
+ * :setting:`FEED_EXPORT_FIELDS`
+ * :setting:`FEED_EXPORT_INDENT`
  * :setting:`FEED_STORAGES`
  * :setting:`FEED_STORAGE_FTP_ACTIVE`
  * :setting:`FEED_STORAGE_S3_ACL`
  * :setting:`FEED_EXPORTERS`
- * :setting:`FEED_STORE_EMPTY`
- * :setting:`FEED_EXPORT_ENCODING`
- * :setting:`FEED_EXPORT_FIELDS`
- * :setting:`FEED_EXPORT_INDENT`
 
 .. currentmodule:: scrapy.extensions.feedexport
 
-.. setting:: FEED_URI
+.. setting:: FEEDS
 
-FEED_URI
---------
+FEEDS
+-----
 
-Default: ``None``
+.. versionadded:: 1.8
+
+Default: ``{}``
 
-The URI of the export feed. See :ref:`topics-feed-storage-backends` for
-supported URI schemes.
+A dictionary in which every key is a feed URI and each value is a nested
+dictionary containing configuration parameters for the specific feed.
+This setting is required for enabling the feed export feature.
 
-This setting is required for enabling the feed exports.
+See :ref:`topics-feed-storage-backends` for supported URI schemes.
 
-.. setting:: FEED_FORMAT
+For instance::
+
+    {
+        'items.json': {
+            'FORMAT': 'json',
+            'ENCODING': 'utf8',
+            'STORE_EMPTY': False,
+            'FIELDS': None,
+            'INDENT': 4,
+        }, 
+        'items.xml': {
+            'FORMAT': 'xml',
+            'FIELDS': ['name', 'price'],
+            'ENCODING': 'latin1',
+            'INDENT': 8,
+        },
+        'items.csv': {
+            'FORMAT': 'csv',
+            'FIELDS': ['price', 'name'],
+        },
+    }
 
-FEED_FORMAT
------------
+The following is a list of the accepted keys and the Scrapy setting that is used
+as a fallback value if that key is not provided for a specific feed definition.
 
-The serialization format to be used for the feed. See
-:ref:`topics-feed-format` for possible values.
+* ``FORMAT``: the serialization format to be used for the feed.
+  See :ref:`topics-feed-format` for possible values. 
+  Mandatory, no fallback setting)
```

Comment:
These bullet points should not end in `)` anymore :slightly_smiling_face:

Reply:
:man_facepalming: